libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.spec.ts

```diff
@@ -203,4 +205,106 @@ describe(WattNavListModule.name, () => {
       expect(link).not.toHaveAttribute('target');
     });
   });
+
+  describe('Supports expandable nav list', () => {
+    it('can set a title on an expandable nav list', async () => {
+      const title = 'Expandable menu';
+
+      const view = await render(
+        `
+        <watt-nav-list [title]="'${title}'" [expandable]="true">
+          <watt-nav-list-item link="/sub-page">Sub page</watt-nav-list-item>
+        </watt-nav-list>
+      `,
+        {
+          imports: [WattNavListModule],
+        }
+      );
+
+      expect(view.queryByText(title)).not.toBeNull();
+    });
+
+    it('supports nav list items in an expandable nav list', async () => {
+      const text = 'Sub page';
+
+      const view = await render(
+        `
+        <watt-nav-list [title]="'Title'" [expandable]="true">
+          <watt-nav-list-item link="/sub-page">${text}</watt-nav-list-item>
+        </watt-nav-list>
+      `,
+        {
+          imports: [WattNavListModule],
+        }
+      );
+
+      expect(view.queryByText(text)).not.toBeNull();
```

Comment:
Its nice to know that its not null, for sure, but can i suggest that you make an expect that checks that the list item is actually set to what we input? :)

Reply:
Isn't that implicitly true by using `view.queryByText(title)`? If the title is not in the DOM, the query will not find it.

Reply:
Isn't the convention that we want to make sure it shows the text that we input? :)

Reply:
Yes, it is for most tests. In this case the focus is to test that we can successfully project content in a component. Not so much about what the projected content is. The content is provided by other developers when they use the component. Other tests in the suite assert that `watt-nav-list-item` is outputting a link, finding it by text value, that it has the right class when active, etc.